src/Listeners/SelectBestAnswer.php

```diff
@@ -59,12 +69,28 @@ public function handle(Saving $event)
             $discussion->best_answer_set_at = Carbon::now();
 
             Notification::where('type', 'selectBestAnswer')->where('subject_id', $discussion->id)->delete();
+            $this->notifyUserOfBestAnswerSet($event);
         } elseif ($id == 0) {
             $discussion->best_answer_post_id = null;
             $discussion->best_answer_user_id = null;
             $discussion->best_answer_set_at = null;
         }
 
-        $this->notifications->delete(new SelectBestAnswerBlueprint($discussion));
+        $this->notifications->delete(new SelectBestAnswerBlueprint($discussion, $this->translator));
+    }
+
+    public function notifyUserOfBestAnswerSet(Saving $event): void
+    {
+        $actor = $event->actor;
+        $bestAnswerAuthoredBy = $this->getUserFromPost($event->discussion->best_answer_post_id);
+
+        $this->notifications->sync(new AwardedBestAnswerBlueprint($event->discussion, $actor, $this->translator), [$bestAnswerAuthoredBy]);
+    }
+
+    public function getUserFromPost(int $post_id): User
+    {
+        $p = Post::where('id', $post_id)->first();
+
+        return User::where('id', $p->user_id)->first();
```

Comment:
This can be simplified to use the `$post->user` Eloquent relationship. There is also a `find` method (I think) that looks for one entry using its ID.

So the following should work:
```php
return Post::find($post_id)->user;
```